src/test/scala/chiseltest/tests/BasicTest.scala

```diff
@@ -26,13 +26,28 @@ class BasicTest extends AnyFlatSpec with ChiselScalatestTester with Matchers {
   }
 
   it should "fail on expect mismatch" in {
-    assertThrows[exceptions.TestFailedException] {
+    assertThrows[ExpectsException] {
```

Comment:
Not sure I agree with this change, I think there's a good reason for tests to look like ScalaTest.
Without the assertThrows, would this show up as a test error or test failure for ScalaTest? I think keeping the distinction is useful.